assets/development-plan.csv

```diff
@@ -1,2 +1,2 @@
 reference,name,description,developmentPlanType,periodStartDate,periodEndDate,developmentPlanGeography,documentationUrl,organisations,entryDate,startDate,endDate,timetableEvents
-XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX,Birmingham New Local Plan,We are working on a new Local Plan for Birmingham which will guide how the city will develop in the future and provide policies to guide decisions on development proposals and planning applications up to 2042.,type,05-Jan-23,05-Jan-23,geography,,Birmingham LPA,05-Jan-23,05-Jan-23,05-Jan-23
\ No newline at end of file
+XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX,Birmingham New Local Plan,We are working on a new Local Plan for Birmingham which will guide how the city will develop in the future and provide policies to guide decisions on development proposals and planning applications up to 2042.,type,05-Jan-23,05-Jan-23,geography,,Birmingham LPA,05-Jan-23,05-Jan-23,05-Jan-23,
```

Comment:
for the csv parsing to work we need to have a commas between all values. Here we were missing it between the value of endDate and the value of timetableEvents (which is "" in this case)